app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,9 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="main_color">#191923</color>
```

Comment:
think about better names
and rename alll to lowercase

Reply:
Done